packages/kbn-es/src/utils/docker.ts

```diff
@@ -443,6 +438,18 @@ export async function printESImageInfo(log: ToolingLog, image: string) {
   log.info(`Using ES image: ${imageFullName} (${revisionUrl})`);
 }
 
+export async function cleanUpDanglingContainers(log: ToolingLog) {
+  log.info(chalk.bold('Cleaning up dangling Docker containers.'));
+
+  try {
+    const { stdout } = await execa('docker', ['container', 'prune', '--force']);
```

Comment:
This may be a little aggressive if the dev wants to keep some other containers around which are stopped, eg dev containers. Could use the same process as `detectRunningNodes` to narrow it down to ES3 nodes, or something like:

`docker ps -a --filter "ancestor=docker.elastic.co/kibana-ci/elasticsearch-serverless:latest-verified"`

Then pass it into `docker rm`